.github/workflows/tests.yml

```diff
@@ -0,0 +1,35 @@
+name: tests
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+# disabled windows due node packages issues
+#        os: [ubuntu-latest, macos-latest, windows-latest]
+        os: [ubuntu-latest, macos-latest]
+        python-version: [ '2.x', '3.x', '3.5', '3.6', '3.7', '3.8', '3.9', 'pypy-2.7', 'pypy-3.6', 'pypy-3.7' ]
+        exclude:
```

Comment:
Could you please add a comment why these specific excludes?

Reply:
i not sure is it make sense to add comments. it was different failures unrelated to js2py. for example missing python or something else missing